lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -0,0 +1,1312 @@
+        'fu_behaviors',
+        'fu_1_symptoms',
+        'fu_2_symptoms',
+        'fu_3_symptoms',
+        'fu_4_symptoms',
+        'fu_1_test',
+        'fu_2_test',
+        'fu_3_test',
+        'fu_4_test',
+        'fu_1_result',
+        'fu_2_result',
+        'fu_3_result',
+        'fu_4_result',
+        'fu_healthy_test',
+        'fu_healthy_result'
+    ]
+
+    date_questions = [
+        'fu_timestamp',
+        'fu_symptom_duration',
+        'fu_date_care',
+        'fu_1_date',
+        'fu_2_date',
+        'fu_3_date',
+        'fu_4_date',
+        'followup_date'
+    ]
+
+    question_categories = {
+        'valueBoolean': boolean_questions,
+        'valueInteger': integer_questions,
+        'valueString': string_questions,
+        'valueDate': date_questions
+    }
+
+    # Combine checkbox answers into one list
+    checkbox_fields = [
+        'fu_care',
+        'fu_which_activites',
+        'fu_behaviors_no',
+        'fu_behaviors_inconclusive',
+        'fu_behaviors',
+        'fu_1_symptoms',
+        'fu_2_symptoms',
+        'fu_3_symptoms',
+        'fu_4_symptoms'
+    ]
+
+    for field in checkbox_fields:
+        record[field] = combine_checkbox_answers(record, field)
+
+    return questionnaire_response(record, question_categories, patient_reference, encounter_reference)
+
+def create_testing_determination_internal_questionnaire_response(record: dict, patient_reference: dict,
+                                            encounter_reference: dict) -> Optional[dict]:
+    """
+    Returns a FHIR Questionnaire Response resource entry for the
+    testing_determination_internal instrument.
+    """
+
+    string_questions = [
+        'testing_trigger',
+        'testing_type'
+    ]
+
+    question_categories = {
+        'valueString': string_questions
+    }
+
+    return questionnaire_response(record, question_categories, patient_reference, encounter_reference, True)
+
+def create_computed_questionnaire_response(record: dict, patient_reference: dict,
+                                            encounter_reference: dict,
+                                            birthdate: datetime, encounter_date: datetime) -> Optional[dict]:
+    """
+    Returns a FHIR Questionnaire Response resource entry for a
+    computed questionnaire.
+    """
+
+    record['age'] = relativedelta(encounter_date, birthdate).years
+
+    integer_questions = [
+        'age'
+    ]
+
+    question_categories = {
+        'valueInteger': integer_questions
+    }
+
+    return questionnaire_response(record, question_categories, patient_reference, encounter_reference)
+
+def create_daily_questionnaire_response(record: dict, patient_reference: dict,
+                                            encounter_reference: dict) -> Optional[dict]:
+    """
+    Returns a FHIR Questionnaire Response resource entry for the daily
+    attestation questionnaire.
+    """
+    boolean_questions = [
+        'daily_symptoms',
```

Comment:
No need to ingest symptoms here since it's already part of encounter.

Reply:
As with the previous set, I chose to ingest these as questionnaire items even though we also capture them as symptoms for convenience.

Reply:
I agree with Jover here that we should not be representing the same exact data in two separate ways within the same FHIR Bundle. Rather, we should continue our practice of choosing the best available FHIR Resource and representing the data that way.

Reply:
I'm no longer ingesting the symptom questions as questionnaire items.